### PR TITLE
Add Vivid clojure-maven-plugin

### PR DESCRIPTION
**Pull request by vivid-inc**
よろしくお願いします

### CODE REVIEW

**Comment by weavejester** [comment]
Ash Ra Template is already in the toolbox.

**Comment by vivid-inc** [comment]
Yes I see how that can be confusing. `boot-art` and `lein-art` stand alone as discrete projects, even though they share the same Git repo as `ash-ra-template`. The "(Ash Ra Template)" portions are now removed from their names; hopefully they are clearly separate tools now.

**Comment by vivid-inc** [comment]
Can't help notice `script/generate.rb` and `index.html.erb`.
Are you open to a PR that switches from ERB to ART? It's the very reason why we invested in developing ART.. ;)
In terms of changes, the code systems will look more or less equivalent, but using Clojure instead of Ruby. And if you prefer, I can recommend a Leiningen-based system using `lein-art`.

**Comment by weavejester** [comment]
> `boot-art` and `lein-art` stand alone as discrete projects, even though they share the same Git repo as `ash-ra-template`. 

Typically I don't separate out Leiningen and Boot tools from their base projects. For example, I include Codox but not lein-codox or boot-codox.

> Are you open to a PR that switches from ERB to ART?

I'm afraid not. I don't see any advantage to switching, and the obvious disadvantage is that Clojure has a greater startup time than Ruby.

**Comment by vivid-inc** [comment]
Applied your indicated pattern. There is now only `clojure-maven-plugin` remaining.

File: projects.yml [hunk 1]
```diff
@@ -4572,3 +4572,9 @@ calva:
   url: https://github.com/BetterThanTomorrow/calva
   categories: [IDE Integration]
   platforms: [clj, cljs]
+
+clojure-maven-plugin:
```
Comment (weavejester) [inline projects.yml:4576]: This should be `clojure_maven_plugin`.

File: projects.yml [hunk 2]
```diff
@@ -4572,3 +4572,9 @@ calva:
...
+  name: vivid:clojure-maven-plugin
```
Comment (weavejester) [inline projects.yml:4577]: This should probably be "Vivid Clojure Maven Plugin" since that's the title of the README.

Comment (vivid-inc) [reply projects.yml]: Alright let's go with your suggestion and see where it takes us

File: projects.yml [hunk 3]
```diff
@@ -4572,3 +4572,9 @@ calva:
   url: https://github.com/BetterThanTomorrow/calva
   categories: [IDE Integration]
   platforms: [clj, cljs]
+
+vivid-clojure-maven-plugin:
```
Comment (weavejester) [inline projects.yml:4576]: You still need to change the `-` to `_`.

Comment (vivid-inc) [reply projects.yml]: あらま

**Pull request merged**